### PR TITLE
[DPE-5028] restart continuous writes

### PR DESCRIPTION
**Pull request by dragomirp**
* Bump charm lib
* Restart continuous writes when in on start hook

### CODE REVIEW

File: tests/integration/helpers.py
Comment (dragomirp) [inline tests/integration/helpers.py:1]: Copied from the PG VM charm

File: src/charm.py [hunk 1]
```diff
@@ -163,9 +159,31 @@ def __init__(self, *args):
...
+    def _on_start(self, event: StartEvent) -> None:
         """Only sets an Active status."""
         self.unit.status = ActiveStatus()
+        if (
+            self.model.unit.is_leader()
+            and PROC_PID_KEY in self.app_peer_data
+            and not self.are_writes_running()
+        ):
+            try:
+                writes = self._get_db_writes()
```
Comment (dragomirp) [inline src/charm.py:179]: We have to get the writes from the db, since we can't consistently persist them

File: src/charm.py [hunk 2]
```diff
@@ -163,9 +159,31 @@ def __init__(self, *args):
...
+            except Exception:
+                logger.debug("Connection to db not yet available")
+                event.defer()
+                return
```
Comment (dragomirp) [inline src/charm.py:183]: PGB VM will take a while to re-establish connection if the principal VM is rebooted.

**Review by marceloneppel — APPROVED** [review]

**Review by taurus-forever — APPROVED** [review]
Thank you!

**Pull request merged**